css/main.css

```diff
@@ -1032,11 +1032,11 @@ input:focus:-ms-input-placeholder { color:transparent;  } /* IE 10+ */
 }
 
 /* Who we are background support for very large screens */
-@media only screen and (min-width: 1620px) {
+@media only screen and (min-width: 1415px) {
   .who-we-are {
     background-image: url(../images/WhoWeAre/who-we-are-no-belt.svg);
     background-position: right;
-    background-size: 1620px;
+    background-size: 1415px;
```

Comment:
We want the background-size to be the same for all screen sizes showing that image so that the height doesn't have to grow.